lib/mayaUsd/fileio/jobs/writeJob.cpp

```diff
@@ -388,7 +490,10 @@ bool UsdMaya_WriteJob::_BeginWriting(const std::string& fileName, bool append)
     }
 
     // Temporarily change Maya's up-axis if needed.
-    _autoAxisChanger = std::make_unique<AutoUpAxisChanger>(mJobCtx.mStage, mJobCtx.mArgs.upAxis);
+    _autoAxisAndUnitsChanger = std::make_unique<AutoUpAxisAndUnitsChanger>(
+        mJobCtx.mStage, mJobCtx.mArgs.upAxis, mJobCtx.mArgs.unit);
+
+    // TODO: handle mJobCtx.mArgs.unit
```

Comment:
Doesn't the command above also handle the units?

Reply:
Ah yes, I will remove this comment in the next PR